tests/units/cli/test_examples/kubernetes-atomicapp/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM projectatomic/atomicapp:0.1.3
+
+MAINTAINER Jason Brooks <jbrooks@redhat.com>
```

Comment:
Going to leave it as is since @jasonbrooks maintains the current kubernetes example. would be good to keep there for reference :)